README.md

```diff
@@ -4,130 +4,180 @@
-After installation, you can follow these [instructions](Usage.md) to try it out.
+let np = Python.import("numpy")
 
-For instructions on building from source, visit
-[apple/swift](https://github.com/apple/swift/tree/tensorflow).
+let myNumpyArray = np.arange([10, 10])  // Create a 10x10 numpy array.
+let myTensor = Tensor(fromNumpyArray: myNumpyArray)  // Seamless integration!
+```
 
 ## Documentation
 
-Below are some documents explaining the Swift for TensorFlow project.
-
-### Conceptual overview
+### Getting started
 
 - [Swift for TensorFlow Design Overview](docs/DesignOverview.md)
 - [Why *Swift* for TensorFlow?](docs/WhySwiftForTensorFlow.md)
+- [Sample Models](https://github.com/tensorflow/swift-models)
+- [Tutorials](https://github.com/tensorflow/swift-tutorials)]
 - [Frequently Asked Questions](FAQ.md)
+- [Swift Tensor API Reference](https://www.tensorflow.org/api_docs/swift/Structs/Tensor)
 
 ### Technology deep dive
 
-- [Graph Program Extraction](docs/GraphProgramExtraction.md)
+The Swift for TensorFlow project builds on top of powerful theoretical
+foundations. For insight into some of the underlying technologies, check
+out the following documentation.
+
+> Beware: the project is moving very quickly, and thus some of these documents
+> are slightly out of date as compared to the current state-of-the-art.
+
 - [Automatic Differentiation Whitepaper](docs/AutomaticDifferentiation.md)
 - [Automatic Differentiation Manifesto](https://gist.github.com/rxwei/30ba75ce092ab3b0dce4bde1fc2c9f1d)
 - [Python Interoperability](docs/PythonInteroperability.md)
 - [Aggregate Parameters and Parameter Update](docs/AggregateParameters.md)
-
-### Swift API reference
-
-- [TensorFlow](https://www.tensorflow.org/api_docs/swift/Structs/Tensor)
-
-### Design proposals
-
-- [One Graph Function per Host Function](proposals/OneGraphFunctionPerHostFunction.md)
+- [Graph Program Extraction](docs/GraphProgramExtraction.md)
 
 ## Source code
 
-The active development of Swift for TensorFlow will happen under the 
-"tensorflow" branch of
-[apple/swift](https://github.com/apple/swift/tree/tensorflow).
-
-These projects include:
+Compiler and standard library development of Swift for TensorFlow happens
+on the `tensorflow` branch of the
+[apple/swift](https://github.com/apple/swift/tree/tensorflow) repository.
 
-- The compiler and standard libraries: [apple/swift](http://github.com/apple/swift/tree/tensorflow)
-- Debugger and REPL support: [apple/swift-lldb](http://github.com/apple/swift-lldb/tree/tensorflow)
+Additional code repositories that make up the core of project include:
 
-Swift for TensorFlow is **not** intended to remain a long-term fork of the official 
-Swift language. New language features will eventually go through the Swift evolution process
-as part of being considered for being pulled into master.
+ - [Debugger and REPL support](http://github.com/apple/swift-lldb/tree/tensorflow)
+ - [High level APIs](https://github.com/tensorflow/swift-apis)
```

Comment:
We've been calling it the "deep learning library" instead of "high-level APIs". "High level" is too heavily overloaded.

Reply:
Yeah, that's much better. I've also switched LLDB to match the new pattern. Thanks!